src/dataflow-types/src/types.rs

```diff
@@ -1138,7 +1138,10 @@ pub struct KafkaSinkConnectorBuilder {
     pub consistency_value_schema: Option<String>,
     pub config_options: BTreeMap<String, String>,
     pub ccsr_config: ccsr::ClientConfig,
-    pub exactly_once: bool,
+    // This makes the sink an exactly_one sink because we need to ensure
```

Comment:
I think we should just get rid of the exactly_once terminology both in internal and user facing documentation. So we could reword this to

```
Forces the sink to always write to the same topic across restarts instead of picking a new topic each time.
```

Reply:
👍